tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -350,21 +469,46 @@ copp/test_copp.py::TestCOPP::test_trap_neighbor_miss:
       - "(asic_type in ['broadcom'] and release in ['202411'])"
       - "(topo_name not in ['t0', 't0-64', 't0-52', 't0-116', 't0-118', 't0-88-o8c80', 't0-isolated-d16u16s1', 't1-isolated-d28u1', 't0-isolated-d32u32s2', 't1-isolated-d56u2'])"
 
-
 #######################################
 #####            crm              #####
 #######################################
+crm/test_crm.py :
+  xfail:
+    reason: "xfail for IPv6-only topologies"
+    conditions:
+      - "'-v6-' in topo_name"
+
 crm/test_crm.py::test_crm_fdb_entry:
   skip:
     reason: "Unsupported topology, expected to run only on 'T0*' or 'M0/MX' topology"
     conditions:
       - "'t0' not in topo_name and topo_type not in ['m0', 'mx']"
 
+crm/test_crm.py::test_crm_route[str4-7060x6-64pe-10-None-ipv4] :
```

Comment:
> str4-7060x6-64pe-10-None-ipv4

The testbed name should be removed from the name, do you mind to help see how we could do it?

Reply:
Since name is part of the match for skip condition, move the check to inside of the test for better clarity and more compatibility.
https://github.com/sonic-net/sonic-mgmt/pull/19641
https://github.com/sonic-net/sonic-mgmt/pull/19606